libcontainer/container_linux.go

```diff
@@ -382,7 +382,7 @@ func (c *linuxContainer) Signal(s os.Signal, all bool) error {
 		return err
 	}
 	// to avoid a PID reuse attack
-	if status == Running || status == Created {
+	if status == Running || status == Created || status == Paused {
```

Comment:
I think to follow runtime-spec standard, we can add a func to transfer runc container state to runtime-spec state.
Please see: https://github.com/opencontainers/runc/pull/1943#issuecomment-443637500